frontend/src/component/common/ToastRenderer/Toast/Toast.styles.ts

```diff
@@ -2,56 +2,42 @@ import { makeStyles } from 'tss-react/mui';
 
 export const useStyles = makeStyles()((theme) => ({
     container: {
-        maxWidth: '450px',
-        background: theme.palette.background.paper,
+        alignItems: 'center',
+        background:
+            theme.mode === 'light' ? '#201E42' : theme.palette.background.paper,
+        borderRadius: theme.shape.borderRadiusMedium,
         boxShadow: theme.boxShadows.popup,
-        zIndex: 500,
+        display: 'flex',
+        flexDirection: 'row',
+        gap: theme.spacing(2),
         margin: '0 0.8rem',
-        borderRadius: '12.5px',
-        padding: '2rem',
-    },
-    innerContainer: {
-        position: 'relative',
+        maxWidth: '450px',
+        padding: theme.spacing(1),
+        paddingInlineStart: theme.spacing(2),
+        zIndex: 500,
+        color: theme.palette.common.white,
     },
     starting: {
         opacity: 0,
     },
-    headerContainer: {
-        display: 'flex',
-        alignItems: 'center',
-    },
-    confettiContainer: {
-        position: 'relative',
-        maxWidth: '600px',
-        margin: '0 auto',
-        display: 'flex',
-    },
-    textContainer: {
-        marginLeft: '1rem',
-        wordBreak: 'break-word',
-    },
     headerStyles: {
+        fontSize: theme.typography.body1.fontSize,
         fontWeight: 'normal',
         margin: 0,
-        marginBottom: '0.5rem',
-    },
-    createdContainer: {
-        display: 'flex',
-        alignItems: 'center',
-        flexDirection: 'column',
     },
     anim: {
         animation: `$drop 10s 3s`,
     },
     checkMark: {
-        width: '65px',
-        height: '65px',
+        marginInlineStart: theme.spacing(1),
```

Comment:
is it to support right-to-left writing mode in this one special case? If not, I'd stick to marginLeft as it's an established pattern in our code and is more familiar/idiomatic in this codebase

Reply:
It's to not have to think about writing modes, actually. If we ever switch (I doubt it), either to right-to-left or top-to-bottom, then this'll still work in the writing direction. So in my eyes, it's just a beefier version of left. I've used it other places, but don't mind switching it out if people hate it.

Reply:
99% of the other code is not prepared for this change so I don't see much value in creating sth that stands out. In a consistent codebase every deviation from a norm is a signal that makes the code reader think. If we switch to margin inlines I'd do it consistently across the board. Otherwise I wouldn't bother